operations/node_management/Disable_Nodes.md

```diff
@@ -48,7 +48,9 @@ Disabling nodes that are not configured correctly allows the system to successfu
     ID = "x5000c1s0b1n2"
     ```
 
-After changing the state of nodes, be cautious when powering them on/off. The preferred method for safely
-powering them on/off is via the Boot Orchestration Service \(BOS\). The Cray Advanced Platform Monitoring
-and Control \(CAPMC\) or Power Control Service \(PCS\) is used to directly control the power for nodes,
-regardless of the state in HSM. PCS/CAPMC does not check if a node is disabled in HSM.
+After changing the state of nodes, be cautious when powering them on/off. The
```

Comment:
Two references to CAPMC in this paragraph

Reply:
Ref OK, having an example bad.